app/src/main/java/org/sopt/dateroad/presentation/ui/navigator/MainActivity.kt

```diff
@@ -4,21 +4,43 @@ import android.os.Bundle
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
 import androidx.activity.enableEdgeToEdge
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.core.splashscreen.SplashScreen.Companion.installSplashScreen
 import dagger.hilt.android.AndroidEntryPoint
+import kotlinx.coroutines.delay
+import org.sopt.dateroad.presentation.ui.splash.SplashScreen
 import org.sopt.dateroad.ui.theme.DATEROADTheme
 
 @AndroidEntryPoint
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         enableEdgeToEdge()
+
+        val splashScreen = installSplashScreen()
+        splashScreen.setOnExitAnimationListener { splashScreenView ->
```

Comment:
```suggestion
       installSplashScreen().setOnExitAnimationListener { splashScreenView ->
```
변수화 하지 말고 이렇게 해주면 좋을 것 같아요 ~